example/components/FilmDetails.tsx

```diff
@@ -61,6 +74,12 @@ export const FilmDetails = ({ filmId, optimize }: Props) => {
     };
   }, [filmId, queryProducers]);
 
+  const connection = useMemo(() => {
+    return data.mapOk((data) => data.film?.characterConnection);
```

Comment:
Given the fact that the connection instance is stable (the `AsyncData` or `Result` are note, sure), could we find a trick to get rid of the `useMemo`?